CHANGELOG.md

```diff
@@ -15,6 +15,36 @@ If you run Hyper-V or want 500 GB, 1 TB or 2 TB VM you can download it from [T&M
 
 PR's are more than welcome. Happy Nextclouding!
 
+## 21.0.3
+
+### Small changelog:
+- Made it possible to add subdomains to deSEC
+- Improved spinner_loading
+- Added dates to automatic updates log
+- Added regular ZFS snapshot prune
+- Added retention for Nextclouds user activities
+- Previewgenerator - allow to clear all previews
+- Update script - update Nextclouds mimetype list
+- Reworked office scripts
+- Update script - change crontab on all installations to 5 minutes
+- Fixed a bug with Netdata
+- Geoblock - updated link to csv file
+- Refactored the bitwarden_mailconfig script
+- Added more functionality to curl_to_dir
+- Docker documentserver - don't restart docker daemon upon installation
+- Restart notify push in some situations
+- Make sure sudo and software-properties-common is installed
+- Fixed password generation in edge cases
+- Reworked the cookielifetime script
+- SMB-server - added option to automatically empty recylce bins
+- SMB-server - added option to empty all recycle bins
+- Minor bugfixes and improvements
+- And more...
+
+### Full changelog:
+- https://github.com/nextcloud/vm/compare/21.0.2..21.0.3
```

Comment:
Hm... I don't think that this will work. Do I need to change this to https://github.com/nextcloud/vm/compare/21.0.2..enh/function-removal-docker-documentserver ?

Reply:
I'm not sure what you mean?

Reply:
What I mean is the compare link
https://github.com/nextcloud/vm/compare/21.0.2..21.0.3

I dont think that it will work because it compares branches.
So I think it would need to be https://github.com/nextcloud/vm/compare/21.0.2..enh/function-removal-docker-documentserver in this case, no?

Reply:
Does this link work? https://github.com/nextcloud/vm/compare/21.0.0..21.0.1

In other words, it will work once it's merged. :)

Reply:
Hm... I don't think so since this branch isn't called 21.0.3

Reply:
Okay, seems like you are not comparing branches but release tags.